Assist users reading the KNX integration page with KNX onboarding (#28406)
home-assistant/home-assistant.github.io: source/_integrations/knx.markdown

- [Time](#time)
- [Weather](#weather)

## Free KNX online training

> For Home Assistant KNX users, you can start a FREE KNX online training & get a discounted ETS Home license [here](https://www.knx.org/knx-en/for-your-home/home-assistant/)

Reviewer: ```suggestion
As a Home Assistant KNX user, you can start a FREE KNX online training and get a discounted ETS Home license on the [KNX website](https://www.knx.org/knx-en/for-your-home/home-assistant/).
```
